### PR TITLE
docs: add more projects using melos

### PR DESCRIPTION
**Pull request by Pavel-Sulimau**
## Description

Let's expand the list of projects using Melos.

## Type of Change

- [ ] ✨ `feat` -- New feature (non-breaking change which adds functionality)
- [ ] 🛠️ `fix` -- Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ `!` -- Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 `refactor` -- Code refactor
- [ ] ✅ `ci` -- Build configuration change
- [x] 📝 `docs` -- Documentation
- [ ] 🗑️ `chore` -- Chore

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -58,6 +58,12 @@ The following projects are using Melos:
 - [GrowERP Flutter ERP](https://github.com/growerp/growerp)
 - [mrverdant13/coverde](https://github.com/mrverdant13/coverde)
 - [woltapp/wolt_modal_sheet](https://github.com/woltapp/wolt_modal_sheet)
+- [cfug/dio](https://github.com/cfug/dio)
+- [simolus3/drift](https://github.com/simolus3/drift)
+- [Lyokone/flutterlocation](https://github.com/Lyokone/flutterlocation)
+- [FlutterGen/flutter_gen](https://github.com/FlutterGen/flutter_gen)
+- [canonical/ubuntu-desktop-installer](https://github.com/canonical/ubuntu-desktop-installer)
```
Comment (spydon) [inline docs/index.mdx:65]: This one is being deprecated, it's better to add the new one instead:
```suggestion
- [canonical/ubuntu-desktop-provision](https://github.com/canonical/ubuntu-desktop-provision)
```

File: packages/melos/README.md [hunk 2]
```diff
@@ -142,6 +142,12 @@ The following projects are using Melos:
 - [mrverdant13/coverde](https://github.com/mrverdant13/coverde)
 - [ThexXTURBOXx/flutter_web_auth_2](https://github.com/ThexXTURBOXx/flutter_web_auth_2)
 - [woltapp/wolt_modal_sheet](https://github.com/woltapp/wolt_modal_sheet)
+- [cfug/dio](https://github.com/cfug/dio)
+- [simolus3/drift](https://github.com/simolus3/drift)
+- [Lyokone/flutterlocation](https://github.com/Lyokone/flutterlocation)
+- [FlutterGen/flutter_gen](https://github.com/FlutterGen/flutter_gen)
+- [canonical/ubuntu-desktop-installer](https://github.com/canonical/ubuntu-desktop-installer)
```
Comment (spydon) [inline packages/melos/README.md:149]:
```suggestion
- [canonical/ubuntu-desktop-provision](https://github.com/canonical/ubuntu-desktop-provision)
```

**Review by spydon — COMMENTED** [review]
Just one comment, otherwise it looks good!

File: docs/index.mdx (hunk 1 above)
Comment (Pavel-Sulimau) [reply docs/index.mdx]: Let's do so, replaced with the new one!

**Review by Salakar — APPROVED** [review]

**Review by spydon — APPROVED** [review]

**Pull request merged**